catalog.html

```diff
@@ -0,0 +1,241 @@
+        <label for="hotel">
+          "Гостиница"
+        </label>
+        <input type="checkbox" name="motel" id="motel" checked>
+        <label for="motel">
+          "Мотель"
+        </label>
+        <input type="checkbox" name="apartments" id="apartments" checked>
+        <label for="apartments">
+          "Апартаменты"
+        </label>
+      </div>
+
+      <div>
+        <p>Стоимость в сутки (Р):</p>
+          <div>
+            <p>
+              "от"
+              <span id="min-price">0</span>
+            </p>
+            <p>
+              "до"
+              <span id="max-price">3000</span>
+            </p>
+          </div>
+        <div>
+          <div id="range-slider">
+            <div data-handle="0" tabindex="0" role="slider" aria-orientation="horizontal" aria-valuemin="0" aria-valuemax="3000" aria-valuenow="0" aria-valuetext="0">
+            </div>
+            <div data-handle="1" tabindex="0" role="slider" aria-orientation="horizontal" aria-valuemin="0" aria-valuemax="4000" aria-valuenow="3000" aria-valuetext="3000">
+            </div>
+          </div>
+        </div>
+              <label for="min-price-value">Минимальная цена</label>
+              <input type="text" id="min-price-value" name="min-price-value" tabindex="-1">
+              <label for="max-price-value">Максимальная цена</label>
+              <input type="text" id="max-price-value" name="max-price-value" tabindex="-1">
+        <button type="submit">Показать</button>
+      </div>
+    </form>
+  </section>
+  <section>
+      <h2>Доступные отели:</h2>
+      <div>
+        <p>
+          "Найдено: "
+          <span id="hotelNumber">3</span>
+        </p>
+        <p>Сортировка:</p>
+        <ul>
+          <li>
+            <a>По цене</a>
+          </li>
+          <li>
+            <a href="#">По типу</a>
+          </li>
+          <li>
+            <a href="#">По рейтингу</a>
+          </li>
+        </ul>
+          <button type="button">
+            <span>Сортировка по убыванию</span>
+          </button>
+          <button type="button">
+            <span>Сортировка по возрастанию</span>
+          </button>
+      </div>
+      <ul>
+        <li>
+          <div>
+            <img src="img/" width="135" height="90" alt="Amara spa">
+              <a href="#">Amara Resort & Spa</a>
+                <p>Гостиница</p>
+                <a href="#">Подробнее</a>
+          </div>
+          <div>
+            <p>От 4000р.</p>
+            <a href="#">Забронировать</a>
+          </div>
+          <ul>
+            <li>
+              <span>1 звезда</span>
+            </li>
+            <li>
+              <span>2 звезды</span>
+            </li>
+            <li>
+              <span>3 звезды</span>
+            </li>
+            <li>
+              <span>4 звезды</span>
+            </li>
+            <li>
+              <span>5 звезд</span>
+            </li>
+          </ul>
```

Comment:
Давай упростим и возьмем звезды из [таблицы юникода](https://unicode-table.com/ru/sets/star-symbols/):
```suggestion
     <span title="5 звезд">★★★★★</span>
```